cotainr/container.py

```diff
@@ -99,6 +99,7 @@ def __enter__(self):
                     "build",
                     "--force",  # sandbox_dir.mkdir() checks for existing sandbox image
                     "--sandbox",
+                    "--fix-perms",
```

Comment:
as we are having this on "by default" how do we turn it off? Shoduld we not instead have "--no-fix-perms" to allow the user to turn it of if they really want?